src/gui/guiFormSpecMenu.cpp

```diff
@@ -3961,6 +4012,11 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 					s.fdefault = L"Changed";
 					acceptInput(quit_mode_no);
 					s.fdefault = L"";
+				} else if ((s.ftype == f_Unknown) &&
+					(s.fid == event.GUIEvent.Caller->getID())) {
+						s.send = true;
```

Comment:
Please swap the indent levels of this code block with the one from the continued `if` condition.